cli/azd/pkg/apphost/aca_ingress_test.go

```diff
@@ -0,0 +1,227 @@
+package apphost
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBuildAcaIngress(t *testing.T) {
+	// Test case 1: Empty bindings
```

Comment:
Possible that the indention hit is not worth it, but if you had used `t.Run` to make these all subtests, I think we could have both used `t.Parallel()` and it would have been easier to run each test under the debugger as an atomic thing.

Reply:
I started from an auto-generated copilot template :)